docs/src/app/components/pages/components/FloatingActionButton/ExampleSimple.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import FloatingActionButton from 'material-ui/lib/floating-action-button';
+import ToggleStar from 'material-ui/lib/svg-icons/toggle/star';
+
+const FloatingActionButtonExampleSimple = () => (
+  <div>
+    <FloatingActionButton>
+      <ToggleStar />
```

Comment:
What do you think about using a `+` icon instead as it's the icon the most used in the google material spec website?
(What would apply to all the examples of the `FloatingActionButton`)